test-prow-e2e.sh

```diff
@@ -13,6 +13,33 @@ function copyArtifacts {
   fi
 }
 
+function waitForAuthOperatorProgressing {
+  output=''
+  count=0
+  sleepDuration=15
+  maxRetries=16  #4 minutes
+  action="started"
+  if [ "$1" == "False" ]; then
+    action="finished"
+    maxRetries=24  #6 minutes
```

Comment:
I'd make this 10 minutes as well (in which case we wouldn't need to change the value from above in this block).

Reply:
@stlaz indicated "It usually takes up to 4 minutes for a pod to reroll...", so 6 minutes seemed like a good timeout, but ok using a default 10 mins for both start and finish.

Reply:
Set to 10 minutes